jupyterlab_wipp/src/index.ts

```diff
@@ -17,103 +16,108 @@ export interface INotebookInfo {
 }
 
 /**
- * Activation function for the jupyterlab_wipp extension.
+ * Initialization data for the jupyterlab_wipp extension.
  */
-function activate(
-  app: JupyterFrontEnd,
-  palette: ICommandPalette,
-  mainMenu: IMainMenu,
-  notebookTracker: INotebookTracker,
-  factory: IFileBrowserFactory,
-  labShell: ILabShell,
-  consoleTracker: IConsoleTracker,
-): void {
-  console.log('JupyterLab extension jupyterlab_wipp is activated!');
-
-  // Show dialogs and register notebooks
-  function registerByPath(path: string): void {
-    // Launch dialog form to collect notebook name and description
-    showDialog(
-      {
-        title: 'Register Notebook in WIPP',
-        body: new NotebookInfoForm()
-      }
-    ).then(
-      result => {
-        if (!result.button.accept) {
-          console.log('Notebook registering cancelled by user.');
-          return;
-        }
-        const info = result.value;
-        console.log(`Form accepted. Name: ${info.name}. Description: ${info.description}`);
-
-        // Launch WippRegister dialog
-        showDialog({
-          title: 'Waiting for WIPP...',
-          body: new WippRegister(path, info.name, info.description, info.openInWipp),
-          buttons: [Dialog.okButton({ label: 'DISMISS' })]
-        })
-      }
-    )
-  }
+const plugin: JupyterFrontEndPlugin<void> = {
+  id: 'jupyterlab_wipp:plugin',
+  autoStart: true,
+  requires: [ICommandPalette, IMainMenu, INotebookTracker, IFileBrowserFactory, ILabShell, IConsoleTracker],
+  activate: (
+    app: JupyterFrontEnd, 
+    palette: ICommandPalette,
+    mainMenu: IMainMenu,
+    notebookTracker: INotebookTracker,
+    factory: IFileBrowserFactory,
+    labShell: ILabShell,
+    consoleTracker: IConsoleTracker
+  ) => {
+    // Run initial health check on backend handlers and check WIPP API is available
+    requestAPI<any>('info')
+      .then(response => {
+        console.log(response.data);
```

Comment:
Remove console.logs

Reply:
This line is equivalent to this line from cookiecutter 
https://github.com/jupyterlab/extension-cookiecutter-ts/blob/057d135d10e62570abcd9b0f1db92022b5ab6faf/%7B%7Bcookiecutter.python_name%7D%7D/src/index.ts#L18
which serves the purpose of announcing that extension was successfully loaded (with additional check of WIPP API being available)